Sources/TokamakDOM/Views/Gestures/_LongPressGestureView.swift

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Szymon on 30/7/2023.
+//
+
+import JavaScriptKit
+import TokamakCore
+import TokamakStaticHTML
+import Foundation
+
+struct _LongPressGestureView<Content: View, G: TokamakCore.Gesture>: View {
+    @State var isPressing = false
+    @State var location: CGPoint = .zero
+    @Binding var gesture: G
+
+    var minimumDuration: Double {
+        if let longPressGesture = gesture as? LongPressGesture {
```

Comment:
Could this be simplified by replacing the generic `G` with `LongPressGesture`, since this View should never be created with another type of gesture.

Reply:
Have not had luck getting around  `Cannot convert value of type 'Binding<G>' to expected argument type 'Binding<LongPressGesture>'`, Any tips & tricks appreciated

Reply:
outdated, structure changed